README.md

```diff
@@ -17,6 +17,30 @@ Git repository centrally stores and serves GDC data models defined in static YAM
 
 ## Update the data models
 
+### Export
```

Comment:
nit: I'm thinking update or sync-models aligns more with the definition than export. Might be graphmanager's export feature messing with my head :)